WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerActivity.java

```diff
@@ -328,23 +328,25 @@ public void doNext(Uri uri) {
 
     private void doMediaIdsSelected(ArrayList<Long> mediaIds, @NonNull PhotoPickerMediaSource source) {
         if (mediaIds != null && mediaIds.size() > 0) {
-            if (mBrowserType == MediaBrowserType.FEATURED_IMAGE_PICKER) {
-                // if user chose a featured image, track image picked event
-                mFeaturedImageHelper.trackFeaturedImageEvent(
-                        FeaturedImageHelper.TrackableEvent.IMAGE_PICKED,
-                        mLocalPostId
-                );
-
+            if (mBrowserType == MediaBrowserType.WP_STORIES_MEDIA_PICKER) {
```

Comment:
is there a reason why the order of the checks was flipped? Seems to have the same results so I'm curious